IdSharp.Tagging/ID3v2/Classes/FrameContainer.cs

```diff
@@ -167,13 +167,14 @@ internal void Read(Stream stream, ID3v2TagVersion tagVersion, TagReadingInfo tag
             }
 
             // Process iTunes comments
+            int iTunesCommentsInsertIndex = m_iTunesCommentsList.Count;
             for (int i = m_CommentsList.Count - 1; i >= 0; i--)
             {
                 IComments comment = m_CommentsList[i];
                 if (comment.Description?.StartsWith("iTun", StringComparison.Ordinal) == true)
                 {
                     m_CommentsList.Remove(comment);
```

Comment:
@ivandrofly I think a forward traversal where we manage the index after removing (and can use normal `Add` on the iTunes Comments) may be more understandable than what I proposed in this PR. Using `RemoveAt` makes sense, thanks.